### PR TITLE
REFACTOR: extract resolveOpDecl()

### PR DESCRIPTION
**Pull request by tsao-chi**

### CODE REVIEW

File: base/src/main/java/org/aya/ref/DefVar.java [hunk 1]
```diff
@@ -61,6 +61,10 @@ private DefVar(Concrete concrete, Core core, @NotNull String name) {
     return new DefVar<>(null, null, name);
   }
 
+  public static OpDecl resolveOpDecl(@NotNull DefVar<?,?> defVar, @NotNull ImmutableSeq<String> moduleName){
+    return defVar.opDeclRename.getOrDefault(moduleName, defVar.opDecl);
+  }
+
```
Comment (imkiva) [inline base/src/main/java/org/aya/ref/DefVar.java:67]: It is not needed to be static. And it would be nice if `@Nullable OpDecl`

**Review by imkiva — APPROVED** [review]
bors d=tsao-chi

**Review by imkiva — COMMENTED** [review]
Should we also change `DefVar.isInfix()`? It is still not checking `opDeclRename`

File: base/src/main/java/org/aya/ref/DefVar.java (hunk 1 above)
Comment (tsao-chi) [reply base/src/main/java/org/aya/ref/DefVar.java]: resolved

**Comment by tsao-chi** [comment]
> ~Should we also change `DefVar.isInfix()`? It is still not checking `opDeclRename`~
> 


We don't have module information in `isInfix` usages

File: base/src/main/java/org/aya/ref/DefVar.java [hunk 2]
```diff
@@ -61,6 +61,10 @@ private DefVar(Concrete concrete, Core core, @NotNull String name) {
     return new DefVar<>(null, null, name);
   }
 
+  public @Nullable OpDecl resolveOpDecl(@NotNull ImmutableSeq<String> moduleName){
```
Comment (ice1000) [inline base/src/main/java/org/aya/ref/DefVar.java:64]: `@NotNull`?

Comment (ice1000) [reply base/src/main/java/org/aya/ref/DefVar.java:64]: @imkiva are you sure it's `@Nullable`?

Comment (ice1000) [reply base/src/main/java/org/aya/ref/DefVar.java:64]: Ok I was stupid

**Comment by ice1000** [comment]
bors r=tsao-chi

**Pull request merged**